frontend/javascripts/admin/voxelytics/task_list_view.tsx

```diff
@@ -558,7 +560,17 @@ export default function TaskListView({
       }}
     >
       <Col xs={10} style={{ display: "flex", flexDirection: "column" }}>
-        <h3 style={{ marginBottom: 0 }}>{readableWorkflowName} </h3>
+        <h3
+          style={{
+            marginBottom: 0,
+            maxWidth: "100%",
+            textOverflow: "ellipsis",
+            overflow: "hidden",
```

Comment:
`hidden` is probably not the best idea. Some other form of text wrapping is preferred to not lose any information.